packages/headline/README.md

```diff
@@ -0,0 +1,21 @@
+# Headline
+
+> A headline with optional standfirst and label
+
+##
+
+```bash
+# Yarn
+$ yarn add @times-visuals/headline
+
+# npm
+$ npm add @times-visuals/headline
+```
+
+## Usage
+
+```jsx
+import Headline from "@times-visuals/headline";
+
+export default () => <Headline headline="This is a headline" />;
```

Comment:
I think this might be a better API:

```suggestion
export default () => <Headline>This is a headline</Headline>;
```

Reply:
Would this mean we'd have three separate packages for headline/standfirst/label?

Reply:
Ah no. Keep the other props the same, so you're full API would be:

```jsx
<Headline label="My label" standfirst="My standfirst">Headline goes here</Headline>
```

Reply:
Ah ok. If I do that how do I access the 'Headline goes here' text in the component itself?

Reply:
Have a look at this, hopefully it'll help: https://codeburst.io/a-quick-intro-to-reacts-props-children-cb3d2fce4891

If not, let me know and I can point you in the right direction

Reply:
Ah thank you. Think I've worked it out and pushed my changes